### DISCUSSION TITLE
What does "--foreground=true: will delete files in the foreground" mean?

### DISCUSSION THREAD

**Discussion by fritzrehde** [kamiyaa/joshuto#216] [Q&A]
I do not understand what is meant with `foreground` in "--foreground=true: will delete files in the foreground". What is the difference between `delete_files` and `delete_files --foreground=true`?

**Comment by D0bhareach** [comment]
It was a typo fixed in [6e968f8.](https://github.com/kamiyaa/joshuto/commit/6e968f8aab4cff7f001c10c3fcbe59030dda6ed0) background=true means it will go to queue of events 
and will be processed in background. Also you can delete to system's trash set `use_trash = true` in
`config/joshuto.toml` for that.

**Reply by CassandraCat** [reply]
The file I deleted was not found in the system Trash
